ivy/functional/frontends/paddle/nn/functional/common.py

```diff
@@ -1 +1,25 @@
 # local
+import ivy
+from ivy.functional.frontends.paddle.func_wrapper import to_ivy_arrays_and_back
+from ivy.func_wrapper import with_unsupported_dtypes
+
+
+@with_unsupported_dtypes({"2.0.1 and below": ("float16", "bfloat16")}, "torch")
```

Comment:
Hello @diksha-shrivastava13 @RickSanchezStoic , in this code block, "torch" is mentioned. Shouldn't it be "paddle" ?

Reply:
It should, actually. I had been closely following PyTorch's implementation. I will open a pull request to refactor the same with 
`@with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")`. Thanks!

Reply:
#16435